Libraries_API.ipynb

```diff
@@ -713,6 +664,151 @@
     "    -X DELETE"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### /query/&lt;library_id&gt;\n",
+    "Add or remove documents the library specified by `library_id` (string) is the unique identifier of the library, as specified in the `/libraries` GET response.\n",
+    "\n",
+    "Functions allowed: add query to a library (POST, GET), delete records from a library (POST).\n",
+    "\n",
+    "#### POST\n",
+    "Add or delete a query to/from a given library. Returns the number of documents added or removed along with the added bibcodes.\n",
+    "\n",
+    "Specify that the request is a POST method by adding `-X POST` to your request. You must include a payload, passed via the `-d` flag, in JSON format. Add this header to your request to indicate your payload is in the appropriate format: `-H \"Content-Type: application/json\"`. The payload must include:\n",
+    "  * query: (json) dict of query parameters as specified in 'search'\n",
+    "  * action: (string) 'add' or 'remove' to add or remove the given bibcodes from the specified library\n",
+    "\n",
+    "*Return data*:\n",
+    "  * number_added: (integer) number of documents added (if 'add' is used)\n",
+    "  * number_removed: (integer) number of documents removed (if 'remove' is used)\n",
+    "  * bibcodes: (list) list of bibcodes added to the library\n",
```

Comment:
Will this list be populated with the removed bibcodes if action=remove? If so, change "added" to "added/removed". Otherwise clarify what will be return if bibcodes are removed